src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './_NoOccasions.scss';
+import { NO_OCCASION_TYPE_MESSAGE } from '../../../../constants/common/commonConstants';
+import Container from '../../../common/Typography/Container';
+import IconCake from '../../../common/Typography/Icons/IconCake';
+import IconRing from '../../../common/Typography/Icons/IconRing';
+
+const OccasionsType = (props) => {
+  return (
+    <Container className="no-occasions-container">
+      <Container className={`no-occasions__icon ${props.iconClass}`}>
+        {props.icon}
+      </Container>
+      <Container className="no-occasions__message">
+        {props.message}
+      </Container>
+    </Container>
+  );
+};
+
+const NoOccasion = (props) => {
+  const { members = [], occasionType } = props;
+  let Occasion;
```

Comment:
This variable is redundant here. Remove it and create a method instead:
```suggestion
const renderNoOccasionContent = () => {
  
    if (occasionType === OCCASION_TYPE .BIRTHDAY) {
      return <OccasionsType icon={<IconCake />} message={NO_OCCASION_TYPE_MESSAGE.BIRTHDAY_MESSAGE} iconClass="birthday-icon" />
    }
    if (occasionType ===  OCCASION_TYPE .ANNIVERSARY) {
      return <OccasionsType icon={<IconRing />} message={NO_OCCASION_TYPE_MESSAGE.ANNIVERSARY_MESSAGE} iconClass="anniversary-icon" />
    }
}
```

Reply:
done